project.clj

```diff
@@ -6,7 +6,7 @@
   :url "http://github.com/threatgrid/ctim"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :pedantic? :abort
+  ;; :pedantic? :abort
```

Comment:
could you uncomment it please?

Reply:
Ah shoot... Good catch... Sorry about that.